data-science-onramp/data-ingestion/setup_test.py

```diff
@@ -118,6 +125,10 @@ def test_setup():
     blob = get_blob_from_path(output_location)
     out = blob.download_as_string().decode("utf-8")
 
+    # check that tables were printed
+    for table_name in TABLE_NAMES:
+        assert table_printed(table_name, out)
```

Comment:
Does this need to be its own function if its only being used once? Can we not do `assert re.search(f"Table {table_name} printed" in out`?